es/introduction.md

```diff
@@ -121,35 +121,35 @@ Si el proceso de auditoría se añade a una codebase existente, las nuevas solic
+* Reducir la deuda técnica y la tasa de fracaso de los proyectos.
+* Tener un mayor control y capacidad de decisión sobre sus sistemas de IT.
+* Mejorar las relaciones con los proveedores con un mejor modelo económico.
 
-[La Foundation for Public Code](https://publiccode.net/) ayuda a las organizaciones públicas a compartir y adoptar software de código abierto, construir comunidades de desarrolladores sostenibles y crear un ecosistema próspero para el código público. Lo hace a través de la tutela o stewardship de codebases. Para este proceso, los administradores de codebases utilizan el Estándar para el Código Público para asegurarse de que el código que supervisan es de alta calidad y se puede mantener en colaboración.
+[La Foundation for Public Code](https://publiccode.net/) ayuda a las organizaciones públicas a compartir y adoptar software de código abierto, construir comunidades de desarrolladores sostenibles y crear un ecosistema próspero para el código público. Lo hace a través de la tutela o dirección de codebases. Para este proceso, los administradores de codebases utilizan el Estándar para el Código Público para asegurarse de que el código que supervisan es de alta calidad y puede ser mantenido de manera colaborativa.
 
 Los usuarios potenciales de las codebases que cumplen con el Estándar para el Código Público pueden esperar que sean altamente reutilizables, fácilmente mantenibles y de alta calidad.
 
 El Estándar para el Código Público hace esto mediante:
 
-* el establecimiento de una terminología común para el desarrollo de código público.
-* estableciendo medidas para ayudar a desarrollar un código público de alta calidad
-* proporcionando orientación sobre cómo cumplir sus criterios y hacer operativa su conformidad
+* El establecimiento de una terminología común para el desarrollo de código público.
+* Estableciendo medidas para ayudar a desarrollar un código público de alta calidad.
+* Proporcionando orientación sobre cómo cumplir sus criterios y hacer operativa su conformidad.
 
 El Estándar para el Código Público pretende ser independiente del tiempo y la tecnología.
 
 ### Para quién es este documento
 
 El Estándar para el Código Público es para aquellas personsa que crean y reúsan código público:
 
-* responsables de políticas
-* personas encargadas de la gestión de equipos y proyectos
-* profesionales del desarrollo de software y el diseño
+* Responsables de políticas.
+* Personas encargadas de la gestión de equipos y proyectos.
+* Profesionales del desarrollo de software y el diseño.
 
 Estas personas trabajan en:
 
-* organizaciones públicas: instituciones y administraciones
-* consultorías y empresas proveedoras de servicios de tecnologías de la información y políticas a organizaciones públicas
+* Organizaciones públicas: instituciones y administraciones.
+* Consultorías y empresas proveedoras de servicios de tecnologías de la información y políticas a organizaciones públicas.
```

Comment:
Capital letters and punctuation signs like the original Standard, although I like your suggestion